packages/bpk-tokens/tokens/base.android.xml

```diff
@@ -24,11 +24,11 @@
   <property name="DURATION_SM" category="animations">200ms</property>
   <property name="DURATION_BASE" category="animations">400ms</property>
   <color name="AUTOSUGGEST_LIST_BACKGROUND_COLOR" category="autosuggest">#ffffffff</color>
-  <color name="AUTOSUGGEST_LIST_ITEM_ACTIVE_BACKGROUND_COLOR" category="autosuggest">#e6e4ebff</color>
-  <color name="AUTOSUGGEST_LIST_ITEM_HIGHLIGHTED_BACKGROUND_COLOR" category="autosuggest">#f3f2f5ff</color>
+  <color name="AUTOSUGGEST_LIST_ITEM_ACTIVE_BACKGROUND_COLOR" category="autosuggest">#ffe6e4eb</color>
```

Comment:
The alpha value moving in these seem like a breaking change. We don't have anyone consuming these values yet though?